config/crd/bases/infrastructure.cluster.x-k8s.io_metal3machinetemplates.yaml

```diff
@@ -58,6 +58,14 @@ spec:
                     description: Spec is the specification of the desired behavior
                       of the machine.
                     properties:
+                      automatedCleaningMode:
+                        default: metadata
+                        description: When set to disabled, automated cleaning will
```

Comment:
See earlier comment. The two lines could be made to be on the same line

Reply:
Please, see my comment above